### PR TITLE
fix: message_to_dict optional fields

### PR DESCRIPTION
**Pull request by legau**

### CODE REVIEW

File: django_socio_grpc/protobuf/json_format.py [hunk 1]
```diff
@@ -1,16 +1,20 @@
-from google.protobuf import descriptor
 from google.protobuf.json_format import MessageToDict, ParseDict
 
 
+def _is_field_optional(field):
+    if not (co := field.containing_oneof):
```
Comment (AMontagu) [inline django_socio_grpc/protobuf/json_format.py:5]: add comment to explain this

File: django_socio_grpc/protobuf/json_format.py [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+        return False
+
+    return len(co.fields) == 1 and co.name == f"_{field.name}"
```
Comment (AMontagu) [inline django_socio_grpc/protobuf/json_format.py:8]: add comment to explain this

File: django_socio_grpc/protobuf/tests/protos/test_proto.proto [hunk 3]
```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```
Comment (AMontagu) [inline django_socio_grpc/protobuf/tests/protos/test_proto.proto:1]: from which serializer this is generated ?

File: django_socio_grpc/protobuf/tests/test_json_format.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+from django_socio_grpc.protobuf.json_format import message_to_dict
+from django_socio_grpc.protobuf.tests.protos import test_proto_pb2
+
+
+def test_message_to_dict():
```
Comment (AMontagu) [inline django_socio_grpc/protobuf/tests/test_json_format.py:5]: add comment to the goal of this test. I don't understand about oneof_field2 that do not appear in this test and not sure if in purpose or not

File: django_socio_grpc/protobuf/tests/protos/test_proto.proto (hunk 3 above)
Comment (legau) [reply django_socio_grpc/protobuf/tests/protos/test_proto.proto:1]: None, written by hand for the tests

**Pull request merged**